src/compiler/scala/tools/nsc/GenericRunnerCommand.scala

```diff
@@ -79,6 +79,7 @@ Other startup options:
 
  -howtorun    what to run <script|object|jar|guess> (default: guess)
  -i <file>    preload <file> before starting the repl
+ -I <file>    preload <file> as though pasted
```

Comment:
to an ordinary user, it's probably not clear what `preload <file> as though pasted` means, and how it behaves differently from `-i` - maybe add something like "i.e., interpreting all at once instead of line-by-line"? you can probably come up with something better..